app/javascript/components/cloud-networks/create-nsxt-cloud-network-form.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import MiqFormRenderer from '@@ddf';
+import createSchema from './create-nsxt-cloud-network-form.schema';
+import { CloudNetworkApi } from '../../utils/cloud-network-api'
+import { ProvidersApi } from '../../utils/providers.api';
+import { handleApiError } from '../../utils/handle-api-error';
+
+class CreateNsxtCloudNetworkForm extends React.Component {
```

Comment:
We're not supposed to implement new components that aren't implemented as functions. Now you can use [react hooks](https://reactjs.org/docs/hooks-reference.html) to create function components with state and side effects.